### PR TITLE
Add 2022q2 CI report

### PR DESCRIPTION
**Pull request by lwhsu**

### CODE REVIEW

File: 2022q2/ci.adoc [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+* Fixed the hardware failure issue of the CI cluster
+
+Work in progress tasks:
+
+* Designing and implementing pre-commit CI building and testing (to support the link:https://gitlab.com/bsdimp/freebsd-workflow[workflow working group])
+* Designing and implementing use of CI cluster to build release artifacts as release engineering does
+* Testing and merging pull requests in link:https://github.com/freebsd/freebsd-ci/pulls[the FreeBSD-ci repo]
+* Reducing the procedures of CI/test environment setting up for contributors and developers
+* Setting up the CI stage environment and putting the experimental jobs on it
+* Organize the scripts in freebsd-ci repository for preparing to merge to src repository
```
Comment (pauamma) [inline 2022q2/ci.adoc:36]: "Organizing" for consistency with other bullet points.
s/for preparing to merge/to prepare for merging/ (reads less stilted to me)
"in the ... repository" x 2 (taste)

File: 2022q2/ci.adoc [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+* Update documents on wiki
```
Comment (pauamma) [inline 2022q2/ci.adoc:37]: s/Update/Updating/

File: 2022q2/ci.adoc [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+Open or queued tasks:
+
+* Collecting and sorting CI tasks and ideas link:https://hackmd.io/@FreeBSD-CI/freebsd-ci-todo[here]
```
Comment (pauamma) [inline 2022q2/ci.adoc:41]: \* link:https://hackmd.io/@FreeBSD-CI/freebsd-ci-todo[Collecting and sorting CI tasks and ideas]

Rationale: https://webaim.org/resources/quickref/#links

File: 2022q2/ci.adoc [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+* Setting up public network access for the VM guest running tests
+* Implementing using bare metal hardware to run test suites
```
Comment (pauamma) [inline 2022q2/ci.adoc:43]: s/using bare metal hardware/use of bare-metal hardware/

File: 2022q2/ci.adoc [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+* Adding drm ports building tests against -CURRENT
+* Planning to run ztest tests
+* Adding more external toolchain related jobs
+* Improving maturity of the hardware lab and adding more hardware under test
```
Comment (pauamma) [inline 2022q2/ci.adoc:47]: By "hardware under test", do you mean "hardware for testing"?

**Review by pauamma — CHANGES_REQUESTED** [review]

**Comment by lwhsu** [comment]
Please feel free to modify more before merging. Thanks!

**Pull request merged**